static/app/components/modals/createReleaseIntegrationModal.tsx

```diff
@@ -0,0 +1,99 @@
+import {Fragment} from 'react';
+
+import {addErrorMessage, addSuccessMessage} from 'sentry/actionCreators/indicator';
+import {ModalRenderProps} from 'sentry/actionCreators/modal';
+import FieldFromConfig from 'sentry/components/forms/fieldFromConfig';
+import Form from 'sentry/components/forms/form';
+import {Field} from 'sentry/components/forms/type';
+import {t, tct} from 'sentry/locale';
+import {Organization, Project} from 'sentry/types';
+import useApi from 'sentry/utils/useApi';
+
+export type CreateReleaseIntegrationModalOptions = {
+  onCancel: () => void;
+  onCreateSuccess: (integration) => void;
+  organization: Organization;
+  project: Project;
+};
+type CreateReleaseIntegrationModalProps = CreateReleaseIntegrationModalOptions &
+  ModalRenderProps;
+
+function CreateReleaseIntegrationModal({
```

Comment:
Not blocking since I know this project is high-priority right now, but I would have liked to see this component generalized as a feature (on the endpoint maybe?) in which you could create an Internal Sentry App with one click via query params, or something. 

This works, but as is though, so I'll make a ticket for this feature as a separate goal.